contribs/gnobro/go.mod

```diff
@@ -12,7 +12,7 @@ require (
 	github.com/charmbracelet/log v0.4.0
 	github.com/charmbracelet/ssh v0.0.0-20240725163421-eb71b85b27aa
 	github.com/charmbracelet/wish v1.4.1
-	github.com/gnolang/gno v0.0.0-00010101000000-000000000000
+	github.com/gnolang/gno v0.2.0
```

Comment:
This seems weird; I don't think it should point to the tag version if using replace. just update it back to v0.0.0 and use mod tidy